lib/reader.ex

```diff
@@ -129,6 +132,17 @@ defmodule Xav.Reader do
     end
   end
 
+  @doc """
+  Seeks the reader to the given time in seconds
+  """
+  @spec seek!(t(), float()) :: t()
+  def seek!(%__MODULE__{reader: ref} = reader, time_in_seconds) do
+    case Xav.Reader.NIF.seek_to_time(ref, time_in_seconds) do
+      {:ok, ref} ->
+        %{reader | reader: ref}
```

Comment:
Is it necessary to update the ref in the struct?

Reply:
Nope! :slightly_smiling_face: